src/components/FeatureSelector/Defaults.tsx

```diff
@@ -35,7 +38,8 @@ const options: FeatureSliderData[] = [
     max: 100,
     currentMin: 0,
     currentMax: 100,
-    units: '%'
+    units: '%',
+    description: 'Instrumentalness predicts whether a track contains no vocals.'
```

Comment:
It looks like this feature is inverted
i.e. 100% intrumentalness means there are vocals...

Could we rename it or invert the values in parsers.tsx

Reply:
it's not inverted its just a confidence rating from spotify
beethoven gave me 80 instrumentalness
rap songs gave me 1 instrumentalness

Reply:
so the documentation is wrong? ok...